src/Workspaces/Remote/Core/Telemetry/ApiUsageIncrementalAnalyzerProvider.cs

```diff
@@ -154,7 +160,7 @@ public async Task AnalyzeProjectAsync(Project project, bool semanticsChanged, In
                     }
 
                     if (symbol.Locations.All(l => l.Kind == LocationKind.MetadataFile) &&
-                        solutionOpt?.GetProject(symbol.ContainingAssembly, cancellationToken) == null)
+                        solution?.GetProject(symbol.ContainingAssembly, cancellationToken) == null)
```

Comment:
I'm curious about the purpose of this analyzer. How is the collected data used?

Reply:
➡️ Discussion moved to #41114